pkg/accumulator/accumulator_interface.go

```diff
@@ -0,0 +1,8 @@
+package accumulator
+
+type AcccumulatorClient interface {
```

Comment:
```suggestion
type AccumulatorClient interface {
```
[Required]: CHECK TYPOS BEFORE YOU PUSH!!!!!!